requirements.txt

```diff
@@ -48,6 +48,8 @@ pytz==2018.3 \
     --hash=sha256:5bd55c744e6feaa4d599a6cbd8228b4f8f9ba96de2c38d56f08e534b3c9edf0d \
     --hash=sha256:887ab5e5b32e4d0c86efddd3d055c1f363cbaa583beb8da5e22d2fa2f64d51ef \
     --hash=sha256:410bcd1d6409026fbaa65d9ed33bf6dd8b1e94a499e32168acfc7b332e4095c0
+newrelic==2.106.1.88 \
```

Comment:
That's now a very ancient version. Latest version is 4.2.0.100

But if you like, we can land this and let Pyup upgrade it. But that might be time consuming too.

Reply:
Good call checking that. I'll bump it here.